numpy/random/setup.py

```diff
@@ -37,9 +37,14 @@ def generate_libraries(ext, build_dir):
 
     EXTRA_LINK_ARGS = []
     EXTRA_LIBRARIES = ['npyrandom']
+    if os.environ.get('CC') == 'xlc':
+        extra_libs = ['xl']
+    else:
+        extra_libs = []
```

Comment:
ok, so two variables related to "extra" libraries is probably excessive--I was just copy-pasting in a rush to test it out